### ISSUE TITLE
zoom buttons disappear and never appear again

### ISSUE THREAD

**Issue by dsmelov** [organicmaps/organicmaps#5194]
**Describe the issue**
Zoom buttons disappear and never appear on a low resolution Android device.

**Steps to reproduce**
1. Buy some strange Android-based device like Caterpillar S22 Flip, Sonim XP3 Plus etc
2. Install organic maps
3. Go to Settings screen and ensure "Zoom buttons visible" option is turned on
4. Go to Map screen, notice Zoom buttons are present
5. Tap anywhere on map, all controls will disappear
6. Tap again, all controls will appear except the Zoom buttons

**Expected behavior**
Zoom buttons should be visible after step 6

**Screenshots**
![IMG_1156](https://github.com/organicmaps/organicmaps/assets/7580271/43652661-9f91-44b3-8da3-7c0754f455ad)
![IMG_1157](https://github.com/organicmaps/organicmaps/assets/7580271/b7dcdacc-bdfa-4317-9d9f-ab351d8d2504)

**System information:**
 - Android 11
 - Organic Maps version: latest
 - Device Model: Cat S22 Flip

**Comment by hoizan** [comment]
It might be due to no GPS  signal

**Comment by arnaudvergnet** [comment]
Damn we did not plan for such small screens :sweat_smile:

When starting the app the zoom buttons are shown without any checks. When you tap on the map to open the information page the buttons move to follow it. When moving we check the available space and if the buttons are too close to the top we hide them so they don't overlap with the compass.

Usually screens are big enough so that when you close the details page there is enough space for the buttons to show back, but this is not the case on such small screens.

We need to decide which minimal screen size we want to support, and if we want to support such cases a possible fix would be to reduce the space between the GPS and the zoom buttons.

**Comment by dsmelov** [comment]
@arnaudvergnet 
Do you really need to perform these checks? IMVHO, zoom buttons are shown perfectly on app start :) Less code - less bugs.
BTW, it would be very nice to have a setting to completely disable controls hiding by tap...

**Comment by arnaudvergnet** [comment]
> Do you really need to perform these checks? IMVHO, zoom buttons are shown perfectly on app start :) Less code - less bugs.

We have to because if you rotate the screen, the compass will appear and we don't want buttons to overlap. If you are in route planning mode there is a header shown and again we don't want buttons to show over it.

Another solution would be to move the compass button somewhere else so it can't overlap with the zoom buttons.

>  BTW, it would be very nice to have a setting to completely disable controls hiding by tap...

Nice suggestion, I'll create an issue for this

**Comment by arnaudvergnet** [comment]
https://github.com/organicmaps/organicmaps/issues/5199
